sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://project.lsst.org/ts/sal_objects/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALTelemetrySet.xsd">
+<SALTelemetry>
+    <Subsystem>CCArchiver</Subsystem>
+    <Version>3.8</Version>
+    <Author></Author>
+    <EFDB_Topic>CCArchiver_sequencerHeartbeat</EFDB_Topic>
+    <item>
+        <EFDB_Name>heartbeat</EFDB_Name>
+        <Description>heartbeat from OCS</Description>
```

Comment:
I'm confused by this description. Are you sending a different heartbeat compared to the SALOBJ one?

Reply:
This was replicated from the ATArchiver, and was in place before I started.  I had presumed this was needed by SALOBJ, but if it's not, I can remove it from here and from ATArchiver.

Reply:
I just spoke to Htut-Khine, and he said the "sequenceHeartbeat" was an artifact of something called the "Sequencer", which apparently doesn't exist anymore?  I can remove that from the CCArchiver and ATArchiver completely.